src/DssSpell.t.sol

```diff
@@ -556,7 +556,7 @@ contract DssSpellTest is DssSpellTestBase {
         assertTrue(spell.done(), "DssSpellTest/spell-not-done");
 
         uint256 nextBalance = dai.balanceOf(conduit);
-        uint256 drawAmount  = 25_000_000 * WAD;
+        uint256 drawAmount  = 0 * WAD;
         assertEq(nextBalance, prevBalance + drawAmount);
```

Comment:
Actually this seems more like it:

```suggestion
        assertEq(nextBalance, prevBalance);
```

However I wonder if this test should not be simply removed instead. You don't typically test for things that will not happen, unless you are enforcing some invariant.

Reply:
yeah normally I wouldn't but since we were explicitly removing that piece of code from the spell, I though in this case it could make sense, but looking at it from how it will look 6 month from now, probably does make more sense to just remove it